MoneroKon D-1 Spam Mitigation Transcription

lh1008 opened the pull request:
Hey @erciccione, 

Transcription for: 
**Spam Mitigation and Blockchain Size Control in Privacy Supporting Permissionless Blockchains**
_Francisco Cabañas, The Monero Project_ 

Author @smeemon 

https://www.monerooutreach.org/monero-konferenco/josh-s.php 

Taiga Instance 213 
Review is being worked by @JOhNKmus 
https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/213

Thank you @smeemon, great work!

erciccione commented:
@lh1008 Did you include @JOhNKmus 's suggestions? is this ready to go?

smeemon commented:
I checked 'approve' on the changes in github. I'll look around a bit
more on github and taiga when I have a bit more time later this week,
but if there's anything more I need to do on this one let me know.
Thanks
On Sun, 2019-08-18 at 07:11 -0700, erciccione wrote:
> @lh1008 Did you include @JOhNKmus 's suggestions? is this ready to
> go?
> 
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> [
> {
> "@context": "http://schema.org",
> "@type": "EmailMessage",
> "potentialAction": {
> "@type": "ViewAction",
> "target": "https://github.com/monero-ecosystem/outreach-docs/pull/185
> ?email_source=notifications\u0026email_token=AMYXC2LI6LRBJTFQX4PRYBTQ
> FFJ7JA5CNFSM4IKWFSS2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2
> ZLOORPWSZGOD4RA3JA#issuecomment-522325412",
> "url": "https://github.com/monero-ecosystem/outreach-docs/pull/185?em
> ail_source=notifications\u0026email_token=AMYXC2LI6LRBJTFQX4PRYBTQFFJ
> 7JA5CNFSM4IKWFSS2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLO
> ORPWSZGOD4RA3JA#issuecomment-522325412",
> "name": "View Pull Request"
> },
> "description": "View this Pull Request on GitHub",
> "publisher": {
> "@type": "Organization",
> "name": "GitHub",
> "url": "https://github.com"
> }
> }
> ]

lh1008 commented:
Hey @erciccione,

Yes, I did include @JOhNKmus suggestions and also made him as co-author in the last commit. This is ready to go.

>:)

erciccione commented on monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Spam_Mitigation_The_Challenge_of_Private_and_Permissionless.txt:
Please remove all the following newlines. Leaving just one would be fine

lh1008 replied:
Okay, I will add something else before pushing. Will do this in a moment.

lh1008 commented:
Okay @erciccione, 

This should be ready to go. Changed the files name, did a final review, found a few things to change and everything is a go. I also added the sites missing text in the final commit.

>:)
